awsshell/wizard.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, session=None, interaction_loader=None,
         self._interaction_loader = interaction_loader
         if interaction_loader is None:
             self._interaction_loader = InteractionLoader()
+        if delegation_loader is None:
```

Comment:
This is a little confusing here especially since a delegation loader could be itself. The docstring may be needed to be updated here. I am not super familiar with the architecture but I am assuming you add the delegation loader if you delegate to a different wizard so if None is provided wouldn't that mean there is no wizard to delegate to?